SendEmailReply add retry for attachment removal (#40788)
demisto/content: Packs/EmailCommunication/ReleaseNotes/2_0_44.md

#### Scripts

##### SendEmailReply

> - Added retry mechanism in SendEmailReply for attachment removal API calls to improve reliability.

Reviewer: ```suggestion
- Added a retry mechanism for attachment removal to improve reliability.
```
